src/lib/userStorage/UserStorageClass.js

```diff
@@ -835,6 +845,8 @@ export class UserStorage {
         receiptHash: get(event, 'data.receiptEvent.txHash'),
         endpoint: {
           address: sender,
```

Comment:
this here is a bug. address should just stay as address. (not address:sender) 
address points to the counter party, so no need for toAddress/fromAddress

Reply:
the address: sender line of code was not made by the WalletChat team, I believe it was made by @johnsmith-gooddollar. The changes we made was to add the toAddress/fromAddress

Reply:
@izuchukwu-eric I understand but in order for it to work you need to fix this bug. replace `address: sender,` with `address,`